static/js/components/EducationDisplay.js

```diff
@@ -21,19 +22,19 @@ import {
 import { userPrivilegeCheck } from '../util/util';
 
 export default class EducationDisplay extends ProfileFormFields {
-  openEditEducationForm = index => {
+  openEditEducationForm: Function = index => {
     openEditEducationForm.call(this, index);
-  }
+  };
 
-  openNewEducationForm = (level, index) => {
+  openNewEducationForm: Function = (level, index) => {
     openNewEducationForm.call(this, level, index);
   };
 
-  deleteEducationEntry = () => {
+  deleteEducationEntry: Function = () => {
```

Comment:
cool, I like having explicit return types so I'll go through and add those to our class properties.